specificatie/verblijfplaats.yaml

```diff
@@ -61,13 +65,16 @@ components:
       description: |
         De officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG) of een verkorte versie.
       type: string
+      pattern: ^[a-zA-Z0-9À-ÿ \.\'\-\']{1,24}?$
```

Comment:
te strak, hier kan ook vanalles ingevuld worden, zoals & / " ( ) + [ ]

Reply:
de datakwaliteit is hier niet altijd zo geweldig, blijkbaar wordt soms een locatiebeschrijving in straatnaam gezet, of "onbekend (VOW)", enz. daar moet de API wel tegen kunnen. Of we moeten daar een andere oplossing voor bedenken (zoals een niet ondersteund teken vervangen door bijv. "?")

Reply:
Ook hier vooralsnog alleen maxLength opgenomen